Simulator/src/main/kotlin/edu/drexel/se575/Block.kt

```diff
@@ -9,7 +11,7 @@ package edu.drexel.se575
  * @property previousBlockHash -- hash of the previous block in the chain
  * @property hash -- sha1 of the timeBlockMinted, previousBlockHash, transactions turned to string and smashed together
  */
-class Block(var transactions: Array<Transaction>, var validator: String,
+@Serializable class Block(var transactions: Array<Transaction>, var validator: String,
```

Comment:
Serializable is reminding me of something. This may help my GSON stuff for the JSON file storage so I may be able to improve that with this little edit